terraform/app/modules/paas/outputs.tf

```diff
@@ -0,0 +1,30 @@
+# Azure Storage Outputs (for future app integration)
+output "documents_azure_storage_account_name" {
+  value     = var.azure_storage_documents_enabled ? module.documents_azure_storage[0].name : null
+  sensitive = false
+}
+
+output "documents_azure_storage_connection_string" {
```

Comment:
the connect_string and access_key can be loaded into secrets
e.g. https://github.com/DFE-Digital/teacher-success/blob/main/terraform/application/application.tf#L27